src/GoogleAdWordsEventForwarder.js

```diff
@@ -53,6 +70,24 @@
 
                 try {
                     if (window.gtag && forwarderSettings.enableGtag == 'True') {
+                        const eventConsentState = getEventConsentState(event);
+
+                        if (eventConsentState) {
+                            var updateConsentPayload = generateConsentStatePayloadFromMappings(
```

Comment:
I think `updated` is more correct.
```suggestion
                            var updatedConsentPayload = generateConsentStatePayloadFromMappings(
```